### ISSUE TITLE
Tasks fight over shared volume

### ISSUE THREAD

**Issue by ralphbean** [konflux-ci/release-service#603]
While watching a long-running release, I noticed that the push-rpms-data-to-pyxis task sat there pending for 20 minutes. I worried something was wrong, so I looked in the events logs and found that it was prohibited from attaching to the pvc that it wanted because another task in the same pipeline was currently using it (the rh-sign-image-cosign task. They would otherwise be able to run in parallel, but one was stopped waiting for the other.

Can anything be done to decrease use or dependence on the shared pvc? Can some tasks use the volume in a read-only capacity that allows multi-attach?

**Comment by johnbieren** [comment]
Very interesting find. Opened https://issues.redhat.com/browse/RELEASE-1291

**Comment by Roming22** [comment]
@ralphbean @johnbieren  While browsing I came across a `# temp workaround until github.com/konflux-ci/release-service/issues/603 is fixed` comment.
It seems the JIRA ticket is closed. Shouldn't the work around/comment be cleaned up now and this issue closed?

**Comment by johnbieren** [comment]
@scoheb we should be good to remove these comments from the rh-advisories pipeline now, right? But probably not close this until all pipelines are migrated to TA?

**Comment by scoheb** [comment]
> [@scoheb](https://github.com/scoheb) we should be good to remove these comments from the rh-advisories pipeline now, right? But probably not close this until all pipelines are migrated to TA?

created https://github.com/konflux-ci/release-service-catalog/pull/946
